src/gengowatcher/runtime.py

```
@@ -0,0 +1,165 @@
+
+
+def run_application(args: argparse.Namespace, console: Console) -> None:
+    """Run the main watcher/TUI application lifecycle."""
+    log, ui_handler = create_logger()
+
+    try:
+        config = AppConfig()
+        configure_logger(log, ui_handler, args, config, tui_enabled=not args.web_only)
+        state = AppState(logger=log)
+        watcher = GengoWatcher(config=config, state=state, logger=log)
+    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)